app/screens/home/recent_mentions/components/mention/mention.tsx

```diff
@@ -84,27 +97,46 @@ function Mention({post, currentUser}: Props) {
         />
     );
 
+    const handlePress = preventDoubleTap(() => {
+        pressDetected.current = true;
```

Comment:
For my understanding, why do we have the `pressDetected` and the `pressTimeout` ?

Reply:
press detected is in case the `preventDoubleTap` finishes before it completes opening the permalink view. the timeout is clearly to clear it, but by being in the context of another function there is not really a need to clear the timeout, 0/5 if you keep the variable for the timeout or not.